_posts/2020-08-28-accesibility-in-mind.md

```diff
@@ -0,0 +1,74 @@
+
+## 2. It improves maintainability
+
+Developers care about maintainability because they wish their designs to be ready for change. A tried and true practice for achieving maintainability is separation of concerns. In the world of front-end development, the main concerns to separate are content and presentation. On occasion, it is unclear where one ends and the other begins. It is there that WCAG helps draw the line. The most fundamental guideline [1.1 - Text Alternatives](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=132%2C211%2C311%2C325%2C131#text-alternatives) mandates:
+
+*All non-text content that is presented to the user has a text alternative that serves the equivalent purpose [...].*
+
+This forces the developer to ponder the content / presentation question. In the case of images, this translates to a decision on whether to use an  [`<img>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img) tag or a [`background-image`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) CSS attribute, making the value of the guideline all the more evident.
+
+## 3. It improves robustness
+
+Web apps today are called upon to achieve [Web Scale](https://en.wikipedia.org/wiki/Scalability). The preeminent architecture for achieving this is Microservices. Microservices however, require that developers adapt the way they put their apps together. The most radical change comes in handling failure states. The Microservices way of doing things is to expect that services might be unavailable and carry on nevertheless. This mindset change is aided by guideline [1.1 - Text Alternatives](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=132%2C211%2C311%2C325%2C131#text-alternatives) (see above). The UI of a modern Microservices-powered web app ought to gracefully fall back to a textual representation when fetching assets fails.
+
+## 4. It improves content quality
+
+Navigating your UI with a screen reader is an eye-opening experience which no front-end developer should spare themselves. The screen reader makes unnecessary repetition painfully obvious.
+
+Lack of clarity in the titles and the labels also sticks out. Uncovering lack of clarity in these crucial pieces of text is of tremendous importance because two pitfalls await the unawary designer: First, the full visual experience often compensates for lack of clarity with context. Danger lurks here because this supporting context might change or even go away. Secondly, the designer is too close to their creation, too close to the business concepts and too involved in the development process to identify lack of clarity. The "fresh eyes" a designer needs to view their UI with is just what a screen reader provides.
```

Comment:
"unawary" -> "unaware" 

This paragraph brings up a great point! It's one that I totally agree with.

Reply:
Done